TradeItIosEmsApi/TradeItConnector.m

```diff
@@ -94,12 +94,12 @@ - (void)getOAuthLoginPopupUrlForMobileWithBroker:(NSString *)broker
                                                                  broker:broker
                                                 interAppAddressCallback:[oAuthCallbackUrl absoluteString]];
 
-    NSMutableURLRequest *request = [TradeItRequestResultFactory buildJsonRequestForModel:oAuthLoginPopupUrlForMobileRequest
+    NSURLRequest *request = [TradeItRequestResultFactory buildJsonRequestForModel:oAuthLoginPopupUrlForMobileRequest
                                                                                emsAction:@"user/getOAuthLoginPopupUrlForMobile"
                                                                              environment:self.environment];
 
-    [self sendEMSRequest:request
-     withCompletionBlock:^(TradeItResult *tradeItResult, NSMutableString *jsonResponse) {
+    [self sendReturnJSON:request
```

Comment:
I extracted this part to Swift. We should move the rest of these functions at some stage, but already I think this change is getting too big.